core/src/test/java/org/elasticsearch/search/aggregations/metrics/geobounds/InternalGeoBoundsTests.java

```diff
@@ -121,7 +121,11 @@ protected InternalGeoBounds mutateInstance(InternalGeoBounds instance) {
             name += randomAlphaOfLength(5);
             break;
         case 1:
-            top += between(1, 20);
+            if (Double.isFinite(top)) {
+                top += between(1, 20);
+            } else {
+                top = randomDouble();
+            }
```

Comment:
Looking at createTestInstance() I don't think so. `top` seems to be the only value that has a special treatment. I think because of the xContent rendering mentioned in the test.